dynamodb_revisions/add_xref_refs.py

```diff
@@ -0,0 +1,95 @@
+"""Add xref reference to current concepts and update ICD-O namespace."""
+import sys
+from pathlib import Path
+from timeit import default_timer as timer
+import click
+from boto3.dynamodb.conditions import Attr
+
+PROJECT_ROOT = Path(__file__).resolve().parents[1]
+sys.path.append(f"{PROJECT_ROOT}")
+
+from disease.database import Database  # noqa: E402
+
+
+def update_identity_records():
+    """Add reference for xref attribute and update ICD-O namespace."""
+    db = Database()
+    batch = db.diseases.batch_writer()
+
+    last_evaluated_key = None
+    while True:
+        if last_evaluated_key:
+            response = db.diseases.scan(
+                FilterExpression=Attr('item_type').eq('identity'),
+                ExclusiveStartKey=last_evaluated_key
+            )
+        else:
+            response = db.diseases.scan(
+                FilterExpression=Attr('item_type').eq('identity')
+            )
+        last_evaluated_key = response.get('LastEvaluatedKey')
+
+        records = response['Items']
+        for record in records:
+            xrefs = {xref for xref in record.get('xrefs', [])}
+
+            # update icdo prefix
+            xrefs_updated = {xref.replace('icd.o', 'icdo')
+                             for xref in xrefs}
+            if xrefs != xrefs_updated:
+                db.update_record(record['concept_id'], 'xrefs',
+                                 list(xrefs_updated))
+
+            # add xref lookup
+            xrefs_l = {xref.lower() for xref in xrefs_updated}
+            for xref in xrefs_l:
+                batch.put_item(Item={
+                    'label_and_type': f"{xref.lower()}##xref",
```

Comment:
Since you called lower above, you don't need to call it again
```suggestion
                    'label_and_type': f"{xref}##xref",
```